oaknational/oak-ai-lesson-assistant
# refactor: demo account to oak components - AI-460

## Description

- [This PR](https://github.com/oaknational/oak-ai-lesson-assistant/pull/204) with mobile bugs fixed
- Some requested design changes:

- Demo user modal centered
- fonts and color font aligned with OAK



## Issue(s)

Fixes #

## How to test

1. Go to https://oak-ai-lesson-assistant-c77tv9szu.vercel-preview.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
